web-filter-base/src/com/untangle/app/web_filter/WebFilterBase.java

```diff
@@ -971,6 +971,21 @@ protected void postInit()
             return;
         }
 
+        /**
+         * If we found older settings do conversion, save, and return
+         * Adding a comment so a change is grabbed
+         */
+        if (readSettings.getVersion() < SETTINGS_CURRENT_VERSION) {
+            // Convert categories
+            List<GenericRule> categories = readSettings.getCategories();
+            addCategory(categories, 85, "Self Harm", "Sensitive", "Graphic content, details around methods of harm, and content that glorifies or promotes suicide, anorexia, bulimia, and other types of self-harm.", true, true, false);
+            addCategory(categories, 86, "DNS Over HTTPS", "Security", "DNS Over HTTPS (DoH) is a network protocol used to communicate domain name server information in an encrypted form over HTTPS, thus hiding the DNS query. Includes URLs of domains performing secure DNS resolution, including DNS Over HTTPS (DoH), DNS Over TLS (DoT), and other secure DNS-related domains and IP addresses.", true, true, false);
+            addCategory(categories, 87, "Low-THC Cannabis Products", "Sensitive", "Sites with content containing low-THC, non-psychoactive products, including CBD oils, resin, extracts, herbs, capsules, supplements, foods, drinks, and toiletries/skin care products. This category may also include other regulated non-psychoactive substances. ", true, false, false);
+            readSettings.setVersion(SETTINGS_CURRENT_VERSION);
+            _setSettings(readSettings);
+            logger.debug("Converted settings: " + this.settings.toJSONString());
```

Comment:
Don't need this.

Reply:
Do you mean just the log? I'm reasonably certain we need to add the new category, set the new settings version, and call _setSettings so that the new categories show in the settings